companion/src/mainwindow.cpp

```diff
@@ -80,8 +80,9 @@ const char * const OPENTX_COMPANION_DOWNLOAD_URL[] = {
 };
 
 MainWindow::MainWindow():
-  downloadDialog_forWait(NULL),
+  downloadDialog_forWait(nullptr),
```

Comment:
why not give the value in the declaration?

Reply:
For the dialog? Because we don't want to create the dialog here... ?  Or do you mean give it a default `nullptr` value in the header?  I'm not sure on the latter honestly... it's a style thing I guess and I haven't really transitioned (yet)... either way is OK, I think, as long as it's consistent per class.  But since complex values can't be initialized in declaration, I tend to stick to constructor for consistency.